src/lib/db/index.ts

```diff
@@ -84,4 +103,16 @@ export default class Database {
       })
       .where({ id: localId })
   }
+
+  insertMatch2 = async (match2: object) => {
```

Comment:
love how it exponentially grows. This is why I would suggest to start utilising models and MVC approach within a TS env. Otherwise we are just duplicating intelli-api

Reply:
https://digicatapult.atlassian.net/browse/L3-52 will sort it?

Reply:
There are better ways of doing this granted. It doesn't grow expontentially though it grows linearly. Sorry minor bugbear there

Reply:
@jonmattgray on that note we are already looking into an ORM and I think this might become a little irrelevant for time being. However, the logic would remain the same. We do lie ORM does, create wrappers for psql bu in our case is a framework we use https://github.com/digicatapult/dscp-matchmaker-api/blob/40774f372de43f1d6f598ad98c95abd0652a673b/src/lib/db/index.ts <- this is an example or the idea of 'factory'